packages/runtime/src/shared/styles.ts

```diff
@@ -104,13 +104,21 @@ export const getIcon = (name: string): SvgIconComponent | undefined => {
 export const getComponentStyles = (
   name: string,
   styles?: React.CSSProperties,
-  returnAsString = true,
+  options: {
+    importantStyles?:
+      | keyof React.CSSProperties
+      | Array<keyof React.CSSProperties>;
+    returnAsString?: boolean;
```

Comment:
Why are these needed?

Reply:
to override these styles of svg
<img width="949" alt="image" src="https://github.com/EnsembleUI/ensemble-react/assets/32386713/c826c78d-1e75-4d21-8571-a5110243d4d3">

Reply:
Can you provide more detail about the problem? Which styles specifically?

You should be able to achieve what you're looking for without these extra properties.